index.html

```diff
@@ -942,13 +943,23 @@ <h3>Processing the manifest</h3>
         To <dfn data-lt="processing a MiniApp manifest">process a MiniApp manifest</dfn>, given [=object=] |json:JSON| and [=ordered map=] |manifest:ordered map|:
```

Comment:
Even the paragraph above explained the extension mechanmism based on APPMANIFEST, it's better to emphasize here that the following steps are executed at the 'extension point' of the APPMANIFEST process algorithm, otherwise, it may reads like a complete process for MiniApp manifest.